DataStructuresMemories/DataStruct.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  DataStruct.swift
+//  DataStructuresMemories
+//
+//  Created by Oleksii Kolomiiets on 4/13/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import Foundation
+
+protocol DataStruct {
+    var sourceOfDataStructuresNames: [String] {get}
```

Comment:
why do we need it in public scope?

Reply:
no we do not need it. I will delete it from protocol and add private property to the model